activerecord/test/cases/batches_test.rb

```diff
@@ -545,6 +545,12 @@ def not_a_post.id
     end
   end
 
+  def test_in_batches_should_error_if_batch_limit_is_nil
+    assert_raise(ArgumentError) do
```

Comment:
Please test the error message here. For such generic errors as `ArgumentError` it is easy to get false positives.

Reply:
test the error message 👍 

```.rb
    error = assert_raise(ArgumentError) do
      Post.in_batches(batch_size: nil) { }
    end
    assert_equal ":of should not be nil", error.message
```
https://github.com/rails/rails/pull/45733/commits/35f5b20f415f0e85842ea9321f8f01c29a595d6b#diff-d7c138a139e1856b3b09ca966d268d40e501bd7974dad612256d4c914730d5a1R549-R552